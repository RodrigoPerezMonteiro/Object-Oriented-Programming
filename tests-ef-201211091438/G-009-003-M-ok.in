2
testG9b.dat
8
3
0
0
