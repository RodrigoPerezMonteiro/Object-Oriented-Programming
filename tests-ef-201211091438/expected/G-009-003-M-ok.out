Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Ficheiro a abrir: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Consultas
1 - Ver pratos vegetarianos disponíveis
2 - Ver clientes sem mensagens de alerta
3 - Ver clientes com encomendas finalizadas
4 - Ver restaurantes com descontos
0 - Sair
Escolha uma opção: OMNIVOROUS|James T. Kirk|kirk@starfleet.net|1|1|32
OMNIVOROUS|Darth Vader|vader@imperial.net|1|4|64
Consultas
1 - Ver pratos vegetarianos disponíveis
2 - Ver clientes sem mensagens de alerta
3 - Ver clientes com encomendas finalizadas
4 - Ver restaurantes com descontos
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 